onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -1006,6 +1011,28 @@ def _where_input_wrangler(
         matcher=lambda sample: sample.kwargs.get("return_indices") is True,
         reason="this aten overload assume return_indices=False",
     ),
+    skip(
+        "nn.functional.max_pool3d",
+        matcher=lambda sample: sample.kwargs.get("ceil_mode") is True
+        and sample.kwargs.get("padding") == 1,
+        reason="this combinations is not supported.",
+    ),
+    skip(
+        "nn.functional.max_pool3d",
+        matcher=lambda sample: sample.kwargs.get("return_indices") is True,
+        reason="this aten overload assume return_indices=False",
+    ),
+    skip(
+        "nn.functional.max_pool3d_with_indices",
```

Comment:
This is blocked by issue: https://github.com/microsoft/onnxruntime/issues/15446. Will update the code once ORT team confirms it is a bug.